src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
I agree with Bruce, but I understand the desire to get this done. Perhaps we could capture these instances and go back and fix them?

---
In reply to: [72540990](https://github.com/dotnet/coreclr/pull/6491#discussion_r72540990) [](ancestors = 72540990,72540635)